roles/addons/ofed/readme.rst

```diff
@@ -20,4 +20,5 @@ Changelog
 ^^^^^^^^^
 
 * 1.0.0: Role creation. Benoit Leveugle <benoit.leveugle@gmail.com>
+* 1.0.1: Set default memlock to unlimited in /etc/security/limits.conf for RDMA access
```

Comment:
We're used to order changelog chronologically inverted (from the most recent to the oldest).